projects-details.html

```diff
@@ -0,0 +1,156 @@
+            <a href="#" class="item">
+              <span class="material-icons-outlined icon">calendar_month</span>
+              <span class="link-txt">Calendar</span>
+            </a>
+          </li>
+
+          <li>
+            <a href="#" class="item">
+              <span class="material-icons-outlined icon">mail</span>
+              <span class="link-txt">Mail</span>
+              <span class="badge warning">8</span>
+            </a>
+          </li>
+
+          <li>
+            <a href="#" class="item">
+              <span class="material-icons-outlined icon">message</span>
+              <span class="link-txt">Chat</span>
+            </a>
+          </li>
+
+          <li>
+            <a href="#" class="item">
+              <span class="material-icons-outlined icon">task</span>
+              <span class="link-txt">Tasks</span>
+            </a>
+          </li>
+
+          <li>
+            <a href="#" class="item active">
+              <span class="material-icons-outlined icon">layers</span>
+              <span class="link-txt">Projects</span>
+            </a>
+          </li>
+
+          <li>
+            <a href="#" class="item">
+              <span class="material-icons-outlined icon">folder</span>
+              <span class="link-txt">Files</span>
+            </a>
+          </li>
+
+          <li>
+            <a href="#" class="item">
+              <span class="material-icons-outlined icon">text_snippet</span>
+              <span class="link-txt">Notes</span>
+            </a>
+          </li>
+
+          <li>
+            <a href="#" class="item">
+              <span class="material-icons-outlined icon">contacts</span>
+              <span class="link-txt">Contacts</span>
+            </a>
+          </li>
+        </nav>
+      </div>
+
+      <div class="content">
+        <header class="app-header">
+          <button class="btn light icon-only app-sidebar-toggle">
+            <span class="material-icons-outlined icon">menu</span>
+          </button>
+
+          <button class="btn light icon-only">
+            <span class="material-icons-outlined icon">search</span>
+          </button>
+
+          <button class="btn light icon-only notification">
+            <span class="material-icons-outlined icon">notifications</span>
+            <span class="dot"></span>
+          </button>
+
+          <span class="divider tall"></span>
+
+          <button class="user-profile-dropdown">
+            <div class="mini-profile">
+              <img
+                src="./images/user4.png"
+                alt="User profile picture"
+                class="avatar round"
+              />
+              <div class="text-info">
+                <span class="title text-ellipsis">John Wick</span>
+              </div>
+            </div>
+
+            <span class="material-icons-outlined icon">arrow_drop_down</span>
+          </button>
+        </header>
+
+        <div class="container projects-details">
```

Comment:
This class name should go with `content`, and here you'll have to name it differently